src/main/java/forestry/arboriculture/genetics/TreeAlyzerPlugin.java

```diff
@@ -87,34 +97,34 @@ public void drawAnalyticsPage1(Screen gui, ItemStack itemStack) {
 			textLayout.newLine();
 
 			{
-				String customPrimaryTreeKey = "trees.custom.treealyzer." + type.getName() + "." + tree.getGenome().getPrimary().getUnlocalizedName().replace("trees.species.", "");
-				String customSecondaryTreeKey = "trees.custom.treealyzer." + type.getName() + "." + tree.getGenome().getSecondary().getUnlocalizedName().replace("trees.species.", "");
+				String customPrimaryTreeKey = "trees.custom.treealyzer." + type.getName() + "." + tree.getGenome().getPrimary().getLocalisationKey().replace("trees.species.", "");
```

Comment:
Will renaming this to `getTranslationKey()` to match vanilla cause obfuscation issues?